t/t2800-jobs-cmd.t

```diff
@@ -1111,6 +1111,27 @@ test_expect_success 'flux-jobs --format={expiration!D:h},{t_remaining!H:h} works
 	test "${expiration}" = "-" &&
 	test "${t_remaining}" = "-"
 '
+# -w, --width tests
+test_expect_success 'flux-jobs -w, --width=N works' '
+	flux jobs -a -w 10 > width10.output &&
+	test_debug "cat width10.output" &&
+	test_must_fail grep -E "^.{11}" width10.output
```

Comment:
grep for "+" sign for truncation? same below?

Reply:
Sure, that can't hurt.